### PR TITLE
Update models to support new functionality.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

**Comment by johngian** [comment]
Please add a comment about the new functionality.

File: woodstock/voting/models.py [hunk 1]
```diff
@@ -4,6 +4,49 @@
...
+        if not self.pk:
+            self.slug = uuslug(self.full_name, instance=self)
+        super(Event, self).save(*args, **kwargs)
+
+
+class Application(models.Model):
+    """Aplication model for Mozillians."""
+    entry_id = models.IntegerField(default=0)
+    number_of_events = models.IntegerField(default=1)
+    event = models.ManyToManyField(Event, through='PreferedEvent')
```
Comment (johngian) [inline woodstock/voting/models.py]: /s/Prefered/Preferred/g/

Comment (johngian) [inline woodstock/voting/models.py]: This also has other occurrences.

File: woodstock/voting/models.py [hunk 2]
```diff
@@ -28,6 +71,11 @@ class MozillianProfile(models.Model):
     avatar_url = models.URLField(max_length=400, default='')
     bio = models.TextField(blank=True, default='')
     username = models.CharField(max_length=100, default='')
+    application = models.ForeignKey(Application,
+                                    blank=True,
```
Comment (johngian) [inline woodstock/voting/models.py]: Move `blank`, `null` to the previous line. It looks like there is some space.

**Comment by johngian** [comment]
Code looks OK.
r+wc
:rocket:

**Pull request merged**